### PR TITLE
Minimal api typedresults sample

### LINKED ISSUE #26370: .NET 7: Code sample for Microsoft.AspNetCore.Http.TypedResults
See [Typed results for minimal APIs](https://devblogs.microsoft.com/dotnet/asp-net-core-updates-in-dotnet-7-preview-4/#typed-results-for-minimal-apis)

Put the code in aspnetcore\fundamentals\minimal-apis\misc-samples\typedResults and remove the Readme.txt file.

### PR DESCRIPTION
**Pull request by sammychinedu2ky**
~Made a new pull request here https://github.com/dotnet/AspNetCore.Docs/pull/26437~

Fixes #26370

### CODE REVIEW

**Comment by 5ryde** [comment]
commit

File: aspnetcore/fundamentals/minimal-apis/misc-samples/typedResults/TypedResultsApiWithTest/Test/WeatherApiTest.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        public void MapWeatherApiTest()
+        {
+            var result = WeatherApi.GetAllWeathers();
+            // assert that the result is a typed result of type WeatherForecast[]
+            Assert.IsInstanceOfType(result, typeof(Ok<WeatherForecast[]>));
+        }
+      
+    }
+
+}
```
Comment (Rick-Anderson) [inline aspnetcore/fundamentals/minimal-apis/misc-samples/typedResults/TypedResultsApiWithTest/Test/WeatherApiTest.cs:19]: Git hub like a blank line at the end of a file
image
```suggestion
}

```

File: aspnetcore/fundamentals/minimal-apis/misc-samples/typedResults/TypedResultsApiWithTest/Program.cs [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    {
+        var forecast = Enumerable.Range(1, 5).Select(index =>
+      new WeatherForecast
+      (
+          DateTime.Now.AddDays(index),
+          Random.Shared.Next(-20, 55),
+          summaries[Random.Shared.Next(summaries.Length)]
+      ))
+      .ToArray();
+        // return a typed result
```
Comment (Rick-Anderson) [inline aspnetcore/fundamentals/minimal-apis/misc-samples/typedResults/TypedResultsApiWithTest/Program.cs:34]:
```suggestion
```

File: aspnetcore/fundamentals/minimal-apis/misc-samples/typedResults/TypedResultsApiWithTest/Test/WeatherApiTest.cs [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+namespace Tests
+{
+    [TestClass()]
+    public class WeatherApiTests
+    {
+        [TestMethod()]
+        public void MapWeatherApiTest()
+        {
+            var result = WeatherApi.GetAllWeathers();
+            // assert that the result is a typed result of type WeatherForecast[]
```
Comment (Rick-Anderson) [inline aspnetcore/fundamentals/minimal-apis/misc-samples/typedResults/TypedResultsApiWithTest/Test/WeatherApiTest.cs:13]:
```suggestion
```

File: aspnetcore/fundamentals/minimal-apis/misc-samples/typedResults/TypedResultsApiWithTest/Program.cs [hunk 4]
```diff
@@ -0,0 +1,42 @@
+var builder = WebApplication.CreateBuilder(args);
+var app = builder.Build();
+
+app.MapGet("/", () => "Hello World!");
+// map the /weatherforecast endpoint to a custom action
+app.MapWeatherApi();
+app.Run();
+
+
+
```
Comment (Rick-Anderson) [inline aspnetcore/fundamentals/minimal-apis/misc-samples/typedResults/TypedResultsApiWithTest/Program.cs:10]:
```suggestion
```

**Review by Rick-Anderson — CHANGES_REQUESTED** [review]
Select "Add suggestion

**Comment by sammychinedu2ky** [comment]
I mistakenly didn't do the add-to batch ... would ensure I use that next time

**Comment by sammychinedu2ky** [comment]
@Rick-Anderson please which extension do you use for linting purposes for C#

**Comment by Rick-Anderson** [comment]
> @Rick-Anderson please which extension do you use for linting purposes for C#

Just what Visual Studio warns. @guardrex  do you use a linter?

**Comment by Rick-Anderson** [comment]
Moved to #26463

**Comment by sammychinedu2ky** [comment]
> > @Rick-Anderson please which extension do you use for linting purposes for C#
> 
> Just what Visual Studio warns. @guardrex do you use a linter?

Ooh alright

**Pull request closed without merging**